solutions/usaco-1084.mdx

```diff
@@ -49,4 +49,60 @@ public class Main {
 
 </JavaSection>
 
+<PySection>
+
+```python
+n = int(input())
+even, odd = 0, 0
+
+cows = [*map(int, input().split())]
+
+for c in cows:
+	if c % 2 == 0: even += 1
+	else: odd += 1
+
+while odd > even:
+	odd -= 2
+	even += 1
+
+if even > odd + 1: even = odd + 1
+print(even + odd)
+```
+
+</PySection>
+
+<CPPSection>
+
+```cpp
+using namespace std;
+#include<iostream>
+#include<vector>
+
+int main() {
+	int even = 0;
+	int odd = 0;
+
+	int n; cin >> n;
+
+	for(int i = 0; i < n; i++) {
+		int cow; cin >> cow;
+		if(cow % 2) odd++;
+		else even++; 
+	}
+
+	while (even < odd) {
+		even += 1;
+		odd -= 2;
+	}
```

Comment:
spaces after control keywords, keep code in new lines, put inline statements in braces